src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -464,11 +474,25 @@ private void the_txout_is_returned()
             txOutResponse.Value.Should().Be(this.transferAmount);
         }
 
+        private void staking_information_is_returned()
```

Comment:
we need another test at some point for when staking is enabled and there are actual coins at stake (the test somewhere near the top of the Specification does startstaking but without any coins at stake) - Perhaps create an Issue to add the test if you agree ?

Reply:
Haven't addressed this with this PR, but could consider in a future one.